kratos/utilities/mortar_utilities.h

```diff
@@ -45,7 +45,7 @@ namespace Kratos
 ///@name  Functions
 ///@{
     
-class MortarUtilities
+class KRATOS_API(KRATOS_CORE) MortarUtilities
```

Comment:
Remove KRATOS_API(KRATOS_CORE). This doesn't need to be exported and will crash on win.

Reply:
I though it was the opposite